tests/e2e/paths/search/granules/granules.spec.js

```diff
@@ -0,0 +1,1244 @@
+import readableGranuleNameGraphQlBody from './__mocks__/readable_granule_name/graphql.body.json'
+import readableGranuleNameGraphQlHeaders from './__mocks__/readable_granule_name/graphql.headers.json'
+import readableGranuleNameTimelineBody from './__mocks__/readable_granule_name/timeline.body.json'
+import readableGranuleNameTimelineHeaders from './__mocks__/readable_granule_name/timeline.headers.json'
+import recurringTemporalGranulesBody from './__mocks__/temporal/recurringTemporalGranules.body.json'
+import sortKeyGranulesBody from './__mocks__/sort_key/granules.body.json'
+import sortKeyGraphQlBody from './__mocks__/sort_key/graphql.body.json'
+import sortKeyGraphQlHeaders from './__mocks__/sort_key/graphql.headers.json'
+import sortKeyTimelineBody from './__mocks__/sort_key/timeline.body.json'
+import sortKeyTimelineHeaders from './__mocks__/sort_key/timeline.headers.json'
+import subscriptionGranulesBody from './__mocks__/subscription/granules.body.json'
+import subscriptionGraphQlBody from './__mocks__/subscription/graphql.body.json'
+import subscriptionGraphQlHeaders from './__mocks__/subscription/graphql.headers.json'
+import subscriptionTimelineBody from './__mocks__/subscription/timeline.body.json'
+import subscriptionTimelineHeaders from './__mocks__/subscription/timeline.headers.json'
+import temporalGranulesBody from './__mocks__/temporal/granules.body.json'
+import temporalGraphQlBody from './__mocks__/temporal/graphql.body.json'
+import temporalGraphQlHeaders from './__mocks__/temporal/graphql.headers.json'
+import temporalTimelineBody from './__mocks__/temporal/timeline.body.json'
+import temporalTimelineHeaders from './__mocks__/temporal/timeline.headers.json'
+import timelineCollectionsBody from './__mocks__/timeline/collections.body.json'
+import timelineCollectionsHeaders from './__mocks__/timeline/collections.headers.json'
+import timelineGranulesBody from './__mocks__/timeline/granules.body.json'
+import timelineGraphQlBody from './__mocks__/timeline/graphql.body.json'
+import timelineGraphQlHeaders from './__mocks__/timeline/graphql.headers.json'
+import timelineTimelineBody from './__mocks__/timeline/timeline.body.json'
+import timelineTimelineHeaders from './__mocks__/timeline/timeline.headers.json'
+import graphQlHeaders from './__mocks__/common/graphql.headers.json'
+
+import { login } from '../../../../support/login'
+
+const defaultCmrPageSize = 20
+
+const testResultsSize = async (page, cmrHits) => {
+  const expectedSize = Math.min(defaultCmrPageSize, cmrHits)
+
+  const metaText = await page.getByTestId('panel-group_granule-results')
+    .getByTestId('panel-group-header__heading-meta-text')
+    .textContent()
+  expect(metaText).toBe(`Showing ${expectedSize} of ${commafy(cmrHits)} matching ${pluralize('granule', cmrHits)}`)
+}
+
+test.describe('Path /search/granules', () => {
+  test.beforeEach(async ({ page }) => {
+    const fakeNow = new Date('2021-06-01').valueOf()
+
+    await page.addInitScript(`{
```

Comment:
This feels like it does not belong here... why is this needed now it wasn't prior to the last commits

Reply:
I realized there was a bug with the previous date code setting code, and it didn't actually work for setting the month properly. It was fortunate that today turned to August and caused the bug to show up. This was the best workaround I could find: https://betterstack.com/community/questions/playwright-mock-date/